### PR TITLE
Parse supplements for series and individual identifiers

### LINKED ISSUE #7: Parse supplements
`ITU-T A Suppl 2`
`ITU-T A Suppl. 2`

### PR DESCRIPTION
**Pull request by mico**
closes #7

### CODE REVIEW

File: spec/pubid_itu/identifier_spec.rb [hunk 1]
```diff
@@ -203,6 +203,26 @@ module Pubid::Itu
       it_behaves_like "converts pubid to pubid"
     end
 
+    context "Supplements" do
+      context "ITU-T H Suppl. 1" do
+        let(:pubid) { "ITU-T H Suppl. 1" }
```
Comment (ronaldtse) [inline spec/pubid_itu/identifier_spec.rb:208]: @mico given a pubid, how can I differentiate between a Series Supplement ("H Suppl.") vs a Document Supplement ("E.156 Suppl.")?

Comment (mico) [reply spec/pubid_itu/identifier_spec.rb:208]: @ronaldtse only by checking of number presence for now.
Now, the difference between `ITU-T H` and `ITU-T H.1` identifiers only in number presence.
Do you have ideas how we can improve it?

Comment (ronaldtse) [reply spec/pubid_itu/identifier_spec.rb:208]: Can you provide sample code to realize the check, and document in README? Thanks.

Comment (mico) [reply spec/pubid_itu/identifier_spec.rb:208]: @ronaldtse I added tests for supplements identifiers parsing and creation. Do you think it would be enough?

**Review by ronaldtse — APPROVED** [review]
Thank you @mico !

**Pull request merged**